src/site/xdoc/releasenotes_old_1-0_5-9.xml

```diff
@@ -721,7 +721,7 @@
         TreeWalker module anymore. For example, the following Checkstyle
         4.4 configuration file:
       </p>
-      <div class="wrapper"><pre class="prettyprint"><code class="language-xml">
+      <div class="wrapper"><pre class="prettyprint"><code class="language-bash">
```

Comment:
Please revert, this is xml.

Please review all your changes yourself, there are bunch of wrong changes